Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md

```diff
@@ -206,7 +213,138 @@ A message created for this error.
 
   logJsonResponse(response);
 
-  graph._drop("myGraph", true);
+  graph._drop("smartGraph", true);
+@END_EXAMPLE_ARANGOSH_RUN
+
```

Comment:
Should there be an explanation for the example here?
All other examples have an explanation in front of them. This one hasn't.

Reply:
Good catch, there should be!